### ISSUE TITLE
connectToStores destroys component's statics.

### ISSUE THREAD

**Issue by sAbakumoff** [yahoo/fluxible#172]
Hello,
I just found that after a component is wrapped by using connectToStores addon:
https://github.com/yahoo/fluxible/blob/master/addons/connectToStores.js
it's missing the statics declared, i.e. the following code will produce the error

**Uncaught TypeError: MyComponent.customMethod is not a function**

```javascript
var connectToStores = require('fluxible/addons').connectToStores;
var MyStore = require('./MyStore');
var MyComponent = React.createClass({
  statics: {
    customMethod: function(foo) {
      return foo === 'bar';
    }
  },
  render: function() {
  }
});
MyComponent= connectToStores(MyComponent, [MyStore], function(stores){
    return stores.MyStore.getState();
});
MyComponent.customMethod('bar');  
```
Is it a bug or feature? :)

**Comment by cesarandreu** [comment]
When you use connectToStores you receive a new component. I noticed this a few weeks back, but decided to just move my static functions and set em on the wrapper component. It'd be nice if they were preserved, though.

I'm doing this:

```jsx
var Component = React.createClass(/* ... */)
Component = connectToStores(/* ... */)
Component.navigateAction = ComponentActions.navigateAction
module.exports = Component
```

I set navigateAction on my components and execute these actions after react-router runs, in my main navigate action by iterating over the routes list that gets passed in the run callback.

It doesn't sound like it'd be hard to fix, although I haven't looked at classes made with `React.createClass`. And ideally, it should work with `React.Component` as well.
